docs/source/notebooks/wps_resolve_rules_demo.ipynb

```diff
@@ -24,19 +24,15 @@
    ],
```

Comment:
<p><code>csv_file.read()</code>  seems simple enough. What are the advantages of creating a handler instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/sandpiper/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I was hoping the handler would prevent the user from having to open and prep the data themselves. Instead they could just pass the path to their file into a handler method and have it take care of it. It would be a pretty tiny method but perhaps just enough to help simplify things in cases where they are multiple files that need prepping.